src/commands/codepush/deployment/lib/date-helper.ts

```diff
@@ -6,7 +6,7 @@ export function formatDate(unixOffset: number): string {
     if (differenceInMinutes(Date.now(), date) < 2) {
       formattedDateString = "Just now";
     } else {
-      formattedDateString = format(date, "MMM DD, hh:mm A");
+      formattedDateString = format(date, "MMM dd, hh:mm a");
```

Comment:
How did you catch this, and is it tested?

Reply:
I tested it by emitting some dates somewhere else. I caught it by reading the doc: https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#200---2019-08-20

Reply:
The behavior of `MMM` also changed but I believe we still want to keep it.

Reply:
Awesome, thanks for the link.